common/repo/repo.proto

```diff
@@ -29,6 +29,7 @@ message File {
     RENAME = 2;
     MODIFY = 3;
     COPY = 4;
+    UNTRACKED = 5;
```

Comment:
Let's not add UNTRACKED, and instead use the other ones (e.g ADD).
I understand that ADD can be tracked or untracked, but that distinction is not important at this point.

Reply:
Done.